depth_first_traversal/DepthFirstTraversal.java

```diff
@@ -0,0 +1,60 @@
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedList;
+import java.util.Set;
+
+public class DepthFirstTraversal {
+
+    public static int[] arr;
+    public static int k = 0;
+    // Array  of lists for Adjacency List Representation
+    public static LinkedList<Integer>[] adj;    
+    
+    //Function to add an edge into the DepthFirstTraversal
+    private static void addEdge(int v, int w) {
+        adj[v].add(w);  
+    }
+
+    // A function used by DFS
+    private static void depthFirstTraversal(int v, Set visited) {
+        // Mark the current node as visited 
+        visited.add(v);
+        arr[k] = v;
+        k++;        
```

Comment:
Just printing the number here would be fine. 
It would allow the end-user to easily remove that one line of code if they don't want to print.